script.js

```diff
@@ -0,0 +1,257 @@
+
+}
+
+console.log(isEven(5));
+
+
+
+//////////////5///////////////////
+
+
+const deleteChars = function (str) {
+   
+    const result = str.slice(1, -1);
+    return result;
+
+}
+
+console.log(deleteChars("Hello"));
+console.log(deleteChars("A"));
+
+
+
+
+const deleteChars = (str) => {
+   
+    const result = str.slice(1, -1);
+    return result;
+    
+}
+
+console.log(deleteChars("Hello"));
+console.log(deleteChars("A"));
+
+
+//////////////6///////////////////
+
+
+const capitalize = function (someWord) {
+   
+    const result = someWord.toLowerCase();
+    return result[0].toUpperCase() + result.slice(1);
+
+}
+
+console.log(capitalize("пиТер"));
+console.log(capitalize("javaScript"));
+
+
+const capitalize = (someWord) => {
+   
+    const result = someWord.toLowerCase();
+    return result[0].toUpperCase() + result.slice(1);
+
+}
+
+console.log(capitalize("пиТер"));
+console.log(capitalize("javaScript"));
+
+
+
+
+//////////////7///////////////////
+
+
+const fn1 = function (mixedArray) {
+    if (Array.isArray(mixedArray)) {
+        const result = [];
+        for (let i = 0; i < mixedArray.length; i++) {
+            if (typeof mixedArray[i] === "number") {
+                const stringedArray = String(mixedArray[i]);
+                result.push(stringedArray);
+            }
+            if (typeof mixedArray[i] === "string") {
+                const numberedArray = +(mixedArray[i]);
+                result.push(numberedArray);
+            }
+        }
+        return result;
+    }
+}
+
+console.log(fn1([1, 2, 3, '4', '5', 6]));
+
+
+const fn1 = (mixedArray) => {
+    if (Array.isArray(mixedArray)) {
+        const result = [];
+        for (let i = 0; i < mixedArray.length; i++) {
+            if (typeof mixedArray[i] === "number") {
+                const stringedArray = String(mixedArray[i]);
+                result.push(stringedArray);
+            }
+            if (typeof mixedArray[i] === "string") {
+                const numberedArray = +(mixedArray[i]);
+                result.push(numberedArray);
+            }
+        }
+        return result;
+    }
+}
+
+console.log(fn1([1, 2, 3, '4', '5', 6]));
+
+
+//////////////8///////////////////
+
+
+const fn1 = function (firstArray, secondArray) {
+    if (Array.isArray(firstArray) && Array.isArray(secondArray)) {
+        const jointArray = [...firstArray, ...secondArray];
+        return jointArray;
+    }
+}
+
+console.log(fn1([1, 2, 3], [4, 5, 6]));
+
+
+const fn1 = (firstArray, secondArray) => {
+    if (Array.isArray(firstArray) && Array.isArray(secondArray)) {
+        const jointArray = [...firstArray, ...secondArray];
+        return jointArray;
+    }
+}
+
+console.log(fn1([1, 2, 3], [4, 5, 6]));
+
+
+
+
+//////////////9///////////////////
+
+const fn1 = function (string, numberOfCharacters) {
+    if (string.length > numberOfCharacters) {
+        const clippedString = string.substring (0, numberOfCharacters);
+        return clippedString + "...";
+        }
```

Comment:
if string lengh will be less than numberOfCharacters your function will return undefined, but I want to receive this string